docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,70 @@
+How to Configure Boot parameters on Ubuntu Core 
+================================================
+
+Different from classic versions of Ubuntu (Desktop and Server), `Ubuntu Core`_ is a minimal, secure and strictly
+confined operating system designed to run on embedded systems, such as IoT devices.
+Ubuntu Core is a snap-based operating system, meaning it uses snap packages to manage applications and services.
+Its read-only nature makes it a very secure and reliable operating system but also makes it a bit more difficult to
+configure certain aspects of the system, such as boot parameters.
+
+
```

Comment:
Explain that this how-to assumes the use of a pre-built image: https://ubuntu.com/core/docs/try-pre-built-images
And that it is necessary to do it on a device and not a VM (for the same reasons specified in other docs).